lib/ui/ui_utils.dart

```diff
@@ -69,3 +69,15 @@ CoreElement createExtensionCheckBox(String extensionName) {
         text,
       ]));
 }
+
+// TODO(kenzie): add hotRestart button.
+
+CoreElement createHotReloadButton() {
```

Comment:
This is a very specific method - unlikely to be used outside of a specific library. Perhaps move this to the library where it'll be used?

Reply:
That library may not yet exist? In which case lets just add a todo to relocate this method later.

Reply:
Added a todo. Likely will use this button in the timeline lib.